src/rapport/04_discussie.Rmd

```diff
@@ -84,3 +113,20 @@ Mogelijke alternatieven:
 -   in dat geval niet werken met stratificatie volgens ecotoop, maar hoogteligging gebruiken om de richting van transecten te bepalen: enkel volgens toenemende hoogte
 -   techniek is echter uitgewerkt voor transecten met vaste lengte, maar de lengte van de raaien en de breedte van de intertidale zone is variabel
 -   de inclusiekansen verhogen evenredig met een maat voor het ecologisch potentieel, maar dan hebben we dus opnieuw onafhankelijke gegevens (kaarten, ...) nodig die dit aangeven.
+
+## Implementatie nieuw steekproefontwerp
+
+De implementatie van een nieuw steekproefontwerp vraagt niet alleen een inspanning van de veldwerkers en de data-gebruikers, het kan ook de vergelijkbaarheid van de nieuwe data met de oude in het gedrang brengen.
+Indien trends over de tijd geanalyseerd worden, is het belangrijk om voorzichtig om te springen met data die niet volgens hetzelfde protocol verzameld werd.
+
+Indien men, zoals eerder gesuggereerd, zou overwegen om het hard substraat minder of niet meer te bemonsteren, is het belangrijk om te weten wat de echte bijdrage van deze zone is aan de biomassa MZB.
+Het is aangeraden om binnen hard substraat minstens één jaar het protocol stikter toe te passen en een random punt met breuksteen niet te verleggen naar een plaats nabij waar er toch wat slik ligt.
+In de jaren nadien bekomt men de biomassa MZB in hard substraat door de oppervlakte hard substraat te vermenigvuldigen met de geschatte biomassa MZB per oppervlaktemaat (bekomen uit een striktere toepassing van de staalnames in hard substraat).
```

Comment:
Of, indien blijkt dat de biomassa in hard substraat verwaarloosbaar is, kan dit stratum verwijderd worden? Mogelijk begrijp ik de zin niet helemaal juist.

Reply:
Dat is eventueel ook een optie indien de biomassa inderdaad heel laag is. Dat is dan aan de "experten" om te beslissen.